.github/stale.yml

```diff
@@ -27,17 +27,18 @@ staleLabel: stale
 
 # Comment to post when marking as stale. Set to `false` to disable
 markComment: >
-  This issue has been automatically marked as stale because it has not had
-  recent activity. It will be closed if no further activity occurs. Thank you
-  for your contributions.
+  Thank you for your contribution! Since there hasn't been any recent activity,
+  this issue has been automatically marked as stale, and may eventually be closed.
+  Please bump this issue by leaving a comment if you want to keep it open.
 
 # Comment to post when removing the stale label.
 # unmarkComment: >
 #   Your comment here.
 
 # Comment to post when closing a stale Issue or Pull Request.
 # closeComment: >
-#   Your comment here.
+  This stale issue is now closed due to inactivity. Please re-open or re-file if
+  if the issue is still relevant. Thank you!
```

Comment:
I think this message is missing the comment `#`s so it'll probably end up being tacked onto the end of the markComment. We also updated the messaging in the vmclarity/kubeclarity repo so there was a bit more info about a) what the criteria for stale was
b) when its likely to be closed.
https://github.com/openclarity/kubeclarity/blob/main/.github/stale.yml#L30-L44

Reply:
Great catch with the comment `#` - that's what I get for quickly copying config from one repo into another :D

I used some of the KubeClarity stalebot messaging as inspiration for creating this PR (see also https://github.com/openclarity/kubeclarity/pull/432). As far as hardcoding the 'number of days' until stale/closed, my concern with baking that into the message text in our `.github` repository default means it's likely someone will forget to update the values in multiple places (the idea with this PR is that we introduce a more aggressive default setting, but encourage maintainers to override with their own values as needed). Thoughts?